src/components/GetImageForm.js

```diff
@@ -26,14 +26,8 @@ export default class GetImageForm extends Component{
     this.setState({camera: e.target.value})
   };
 
-
-  componentDidMount() {
```

Comment:
`componentDidMount` is for things that we want to happen after a component is mounted. While we might want to call fetch after the component is mounted, we also want to call it when a button is clicked, so it's better to abstract this fetching functionality as a method that can be called when we need it.